ext/zstdruby/zstdruby.c

```diff
@@ -79,15 +79,23 @@ static VALUE decompress_buffered(const char* input_data, size_t input_size)
   return output_string;
 }
 
-static VALUE decompress(VALUE self, VALUE input)
+static VALUE decompress(int argc, VALUE *argv, VALUE self)
 {
+  VALUE input = argv[0];
```

Comment:
`rb_scan_args` is better for parsing args.
How about the below way?

```
VALUE input;
VALUE continuous_decompression_value;
rb_scan_args(argc, argv, "11", & input, &continuous_decompression_value);

VALUE continuous_decompression;
if (NIL_P(continuous_decompression_value)) {
    continuous_decompression = RUBY_T_FALSE;
}
```

Reply:
`RUBY_T_FALSE` is class type. I think you should use `Qtrue` or `Qfalse` instead in this case.